app/classifier/tasks/survey/index.cjsx

```diff
@@ -48,7 +48,8 @@ module.exports = React.createClass
       {if @state.selectedChoiceID is ''
         <Chooser task={@props.task} filters={@state.filters} onFilter={@handleFilter} onChoose={@handleChoice} onRemove={@handleRemove} annotation={@props.annotation} focusedChoice={@state.focusedChoice} />
       else
-        <Choice task={@props.task} choiceID={@state.selectedChoiceID} onSwitch={@handleChoice} onCancel={@clearSelection} onConfirm={@handleAnnotation} />}
+        existingAnnotation = _.find(@props.annotation.value, 'choice', @state.selectedChoiceID) || {}
```

Comment:
`_.find` returns the first match https://lodash.com/docs/4.17.4#find
In hindsight, `.filter` is better for this.